### ISSUE TITLE
Medusa no longer snatches any *new* torrents at all, automatic or manually

### ISSUE THREAD

**Issue by HybridEyeGrillage** [pymedusa/Medusa#5781]
**Describe the bug**
For several weeks now, I've been unable to get any shows updated or any new shows snatched via Medusa. I constantly receive the same type of error for any shows; be it an old, continuing, or newly added one whenever the daily backlog runs or doing manual searches.

**To Reproduce**
1. Go to show list and select a show.
2. Click on Manual search icon next to Season name.
3. Click on desired torrent(s) via the snatch icon, see a red 'X'.
4. Click on View Log from dropdown and see error(s) in logs about `IndexError: list index out of range` for torrent selected.

**Expected behavior**
I would expect that the shows would be polled for new updates via the search providers that are selected (mostly the free ones without a needed user login or so). Even when I do a manual search through on show itself, it errors when trying to snatch it with the same issues, same lines, and same IndexError; doesn't matter if it's an anime show or a regular television show.

**Logs:**
<details>
```
2018-11-25 10:54:21 ERROR    SNATCHQUEUE-SNATCH-72454 :: [54da0c7] Snatch failed! For result: [HorribleSubs].Detective.Conan.-.922.[720p].mkv
Traceback (most recent call last):
  File "C:\Medusa\Medusa\medusa\search\queue.py", line 502, in run
    self.success = snatch_episode(result)
  File "C:\Medusa\Medusa\medusa\search\core.py", line 167, in snatch_episode
    result_downloaded = client.send_torrent(result)
  File "C:\Medusa\Medusa\medusa\clients\torrent\generic.py", line 246, in send_torrent
    if not self._get_auth():
  File "C:\Medusa\Medusa\medusa\clients\torrent\utorrent_client.py", line 85, in _get_auth
    self.auth = re.findall('<div.*?>(.*?)</', self.response.text)[0]
IndexError: list index out of range
```
</details>

**Screenshots**
https://www.dropbox.com/s/ys0ul8xb6744yjf/Screenshot%202018-11-25%2011.18.38.png?dl=0

**Medusa (please complete the following information):**
Python Version: 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)]
Operating System: Windows-10-10.0.17134
Locale: cp1252
Branch: master
Database: 44.12
Commit: a1d899b
Link to Log: https://gist.github.com/d1b6ddc1773f08e3c38f8ca148880f37

**Additional context**
When I first reported this issue, it got merged into [this one](https://github.com/pymedusa/Medusa/issues/3860). @medariox had asked me about my utorrent version (3.4.9_pro), but I don't believe that's the root of the issue as it worked flawlessly before and still does with other FOSS like Radarr.

**Comment by medariox** [comment]
Are you able to access `http://192.168.178.15:8085/gui/token.html` from your browser?

**Comment by HybridEyeGrillage** [comment]
@medariox I get prompted for a username/password (pre-saved). After that, I just have a white screen.

**Comment by medariox** [comment]
Make sure you entered the correct credentials in the uTorrent settings and that they are saved in your `config.ini`.

**Comment by HybridEyeGrillage** [comment]
@medariox Verified that are correct in uTorrent, in Medusa (even other programs that use it), and Medusa/data/config.ini.

**Comment by HybridEyeGrillage** [comment]
@medariox I removed all the info for my downloader, disabled the feature, restarted it and enabled it again. I filled the information in again (just as before) and now it seems to be working again. It's rather strange because I've changed nothing in my settings (router or Medusa). I also noticed that when trying to test it, it never gave any feedback that it was unable to connect or able to connect- maybe there's some other logging or error handling missing from detecting that issue?
